meta-signing-key/classes/user-key-store.bbclass

```diff
@@ -536,10 +536,6 @@ def boot_sign(input, d):
     status, output = oe.utils.getstatusoutput(cmd)
     if status:
         bb.fatal('Failed to sign: %s' % (input))
-    gpg_conf = bb.utils.which(os.getenv('PATH'), 'gpgconf')
-    cmd = 'GNUPGHOME=%s %s --kill gpg-agent' % \
-            (gpg_path, gpg_conf)
-    status, output = oe.utils.getstatusoutput(cmd)
```

Comment:
So how do you plan to deal with the gpg-agent daemon's life cycle management?

Reply:
I plan to add a lock or release it at the end of do image.
do you have any suggestion about this?

Reply:
So please put the new solution here rather than a simple revocation.

Reply:
OK, will send a new patch to fix this issue.